### PR TITLE
Add allergens in Food provider

### PR DESCRIPTION
**Pull request by panilya**

### CODE REVIEW

File: src/test/java/net/datafaker/providers/foods/FoodTest.java [hunk 1]
```diff
@@ -21,6 +21,7 @@ void measurement() {
     protected Collection<TestSpec> providerListTest() {
         return List.of(
             TestSpec.of(food::dish, "food.dish"),
+            TestSpec.of(food::allergen, "food.allergens"),
```
Comment (snuyanzin) [inline src/test/java/net/datafaker/providers/foods/FoodTest.java:24]: I would prefer abc order

**Comment by snuyanzin** [comment]
in general lgtm 
there is a nit comment

File: src/test/java/net/datafaker/providers/foods/FoodTest.java (hunk 1 above)
Comment (panilya) [reply src/test/java/net/datafaker/providers/foods/FoodTest.java:23]: Fixed

**Review by kingthorin — APPROVED** [review]
Thanks

**Pull request merged**